sale_double_validation/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{'name': 'Double validation for Sales',
+ 'summary': "",
+ 'version': '10.0.1.0.0',
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'category': 'sale',
+ 'depends': [
+     'sale',
+     'sales_team',
+ ],
+ 'website': 'www.camptocamp.com',
```

Comment:
@leemannd IMO change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

Reply:
@nikul-serpentcs You're right. Rules have changed since the first commit of the PR. Updating it